<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionTrajet</class>
 <widget class="QDialog" name="gestionTrajet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trajet</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Gestion des trajets</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>721</width>
       <height>471</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>ajouter un trajet</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>numerotrajet</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>station de depart</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>station d arrivee</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_num">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>80</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_stationd">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>130</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_stationa">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>180</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>290</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>heure de depart</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_dated">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>240</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>heure d arrivee</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_datea">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>299</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>731</width>
         <height>411</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxIDBus_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>360</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID Bus</string>
       </property>
      </widget>
      <zorder>label_8</zorder>
      <zorder>label</zorder>
      <zorder>label_2</zorder>
      <zorder>label_3</zorder>
      <zorder>lineEdit_num</zorder>
      <zorder>lineEdit_stationd</zorder>
      <zorder>lineEdit_stationa</zorder>
      <zorder>pb_ajouter</zorder>
      <zorder>label_5</zorder>
      <zorder>lineEdit_dated</zorder>
      <zorder>label_6</zorder>
      <zorder>lineEdit_datea</zorder>
      <zorder>comboBoxIDBus_2</zorder>
      <zorder>label_19</zorder>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Afficher les trajets</string>
      </attribute>
      <widget class="QTableView" name="tabtrajet">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>711</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Supprimer un trajet</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_numt">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>numero du trajet</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>240</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>731</width>
         <height>411</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_9</zorder>
      <zorder>lineEdit_numt</zorder>
      <zorder>label_4</zorder>
      <zorder>pb_supprimer</zorder>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>modifier un treajet</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_stationam">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>station d arrivee</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_dateam">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>240</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_datedm">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>numero du trajet</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date de depart </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_numm">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date d arrivee</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>station de depart</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_stationdm">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>300</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>731</width>
         <height>411</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxIDBus">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>300</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID Bus</string>
       </property>
      </widget>
      <zorder>label_10</zorder>
      <zorder>lineEdit_stationam</zorder>
      <zorder>label_12</zorder>
      <zorder>lineEdit_dateam</zorder>
      <zorder>lineEdit_datedm</zorder>
      <zorder>label_13</zorder>
      <zorder>label_14</zorder>
      <zorder>lineEdit_numm</zorder>
      <zorder>label_15</zorder>
      <zorder>label_16</zorder>
      <zorder>lineEdit_stationdm</zorder>
      <zorder>pb_modifier</zorder>
      <zorder>comboBoxIDBus</zorder>
      <zorder>label_18</zorder>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>recherche alphabetique</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>390</y>
       <width>231</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>recherche alphabetiqe</string>
     </property>
    </widget>
    <widget class="QTableView" name="tablea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>591</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>731</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_11</zorder>
    <zorder>pushButton</zorder>
    <zorder>tablea</zorder>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>recherche</string>
    </attribute>
    <widget class="QTableView" name="tabler">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>601</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>saisir le numero du tarje a rechercher :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_numr">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>731</width>
       <height>501</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_17</zorder>
    <zorder>tabler</zorder>
    <zorder>label_7</zorder>
    <zorder>lineEdit_numr</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
